### ISSUE TITLE
query and filter on inherited properties (language/site/...)

### ISSUE THREAD

**Issue by ComLock** [enonic/xp#6285]
Hi

A content have a number of inherited properties.

I can make a query for site:
```(_path LIKE '/content/no/*' or _path LIKE '/content/se/*')```

But I don't think its possible to filter on it.

I don't know if its possible to query by inherited language unless:
one site = one language

Perhaps site and language should be implemented similar to how permissions are implemented?
Though a content inherits its permissions, the content itself actually contains the inherited permission...

**Comment by mortyeriksen** [comment]
Good point. This needs to be considered when we improve the language handling in XP. Always remember to set language on your site, then all child items will inherit this.

**Comment by mortyeriksen** [comment]
We will also create a task to make language mandatory for sites for 7.0.
